src/Engine/Network/UDPServer.cpp

```diff
@@ -12,34 +12,120 @@ UDPServer::UDPServer(int port)
 
 UDPServer::~UDPServer()
 { }
-
-void UDPServer::Send(Packet& packet, PlayerDefinition & playerDefinition)
+// TODO: Fix correct groups
+void UDPServer::Send(Packet& packet, PlayerDefinition& playerDefinition)
 {
     packet.UpdateSize();
     try {
-        int bytesSent = m_Socket->send_to(
-            boost::asio::buffer(packet.Data(), packet.Size()),
-            playerDefinition.Endpoint,
-            0);
-        LOG_INFO("Size of packet is %i", bytesSent);
+        //Debug
+        int debugTheSixeOfpacket = packet.Size();
```

Comment:
The variable is misspelled and unused, comment/remove it, or spell it correctly at least.